internal/impl/sentry/processor_capture.go

```diff
@@ -0,0 +1,348 @@
+package sentry
+
+import (
+	"context"
+	"fmt"
+	"runtime/debug"
+	"strings"
+	"time"
+
+	"github.com/benthosdev/benthos/v4/public/bloblang"
+	"github.com/benthosdev/benthos/v4/public/service"
+	"github.com/getsentry/sentry-go"
+)
+
+const (
+	transportAsync = "async"
+	transportSync  = "sync"
+)
+
+func getBenthosVersion() string {
+	bi, ok := debug.ReadBuildInfo()
```

Comment:
I know it's gross but for now the best place to grab a version is probably from cli.Version directly: https://github.com/benthosdev/benthos/blob/main/internal/cli/run.go#L27, walking deps for module names unfortunately only seems to work for custom benthos builds where the module is imported separately, unless they fixed that recently.

Reply:
make sense. updated!